services/storage/tests/unit/test__worker_tasks_paths.py

```diff
@@ -188,9 +198,10 @@ async def test_path_compute_size(
             selected_node_s3_keys
         )
         accumulated_subfolder_size += await _assert_compute_path_size(
```

Comment:
Here using `assert` for naming is OK since this is a test and inside you are actually using `assert`